zuul.d/project-templates.yaml

```diff
@@ -595,21 +595,15 @@
     name: ansible-collections-vyos-vyos
     check:
       jobs:
```

Comment:
we should add an yaml anchor, so we can also ensure the same jobs are in gate.  As it stands, gate do not have local jobs now